code/datums/outfit/civilian.dm

```diff
@@ -103,10 +103,7 @@
 	)
 
 	items_to_collect = list(
-		/obj/item/ammo_casing/shotgun/beanbag = SURVIVAL_BOX,
```

Comment:
Technical detail : because I'm looping over keys, and not over elements of the list, `/obj/item/ammo_casing/shotgun/beanbag` was only considered (and hence spawned) once.